ekncontent/Makefile.am

```diff
@@ -85,8 +86,20 @@ ekncontent_sources = \
 	ekncontent/eknc-video-object-model.c \
 	$(NULL)
 
+glib_enum_headers = \
```

Comment:
Needed for the drop in "add enum support" makefile, which I'd like to keep unaltered.